src/main/frontend/components/page.cljs

```diff
@@ -308,6 +342,14 @@
         (when (and (not sidebar?)
                    (not block?))
           [:div.flex.flex-row.space-between
+           (when (mobile-util/is-native-platform?)
+            [:div.flex.flex-row.pr-2
```

Comment:
`items-center` in `flex` should work. But there's a:

```css
h1.title {
  margin-bottom: 1.5rem;
....
}
```

So the control arrow cannot be aligned with the center of title.

Reply:
22pt here is an estimated value which makes the controller center likely. Yes, as you said, it is not accurate for center aligning.